android/build.gradle

```diff
@@ -17,16 +17,18 @@ buildscript {
             url 'https://maven.fabric.io/public'
         }
     }
+    
+    def taskRequests = getGradle().getStartParameter().getTaskRequests().toString()
+    def isPlay = !(taskRequests.contains("Foss") || taskRequests.contains("foss"))
```

Comment:
I guess is better if you use `toString().toLowerCase()` on `getTaskRequests()` , so any variant of `Foss` will be considered a foss build.
Change this on all necessary places.